packages/raven-js/typescript/raven.d.ts

```diff
@@ -145,7 +145,7 @@ declare namespace Raven {
         * @param {object} options Optional set of global options [optional]
         * @return {Raven}
         */
-    config(dsn: string, options?: RavenOptions): RavenStatic;
+    config(dsn: string | false | null | undefined, options?: RavenOptions): RavenStatic;
```

Comment:
I didn't make the parameter optional since calling `config()` without anything felt weird to me. Disabling raven is probably often due to passing a value that is conditionally falsy.